src/utils/image/compress/index.ts

```diff
@@ -1,13 +1,17 @@
 import {Image} from 'react-native-compressor';
 
 const compress = async (imagePath: string, type: 'base64' | 'uri' = 'uri') => {
-  const compressedImage = await Image.compress(imagePath, {
-    compressionMethod: 'auto',
-    input: type,
-    returnableOutputType: type
-  });
+  try {
+    const compressedImage = await Image.compress(imagePath, {
+      compressionMethod: 'auto',
+      input: type,
+      returnableOutputType: type
+    });
 
-  return compressedImage;
+    return compressedImage;
+  } catch (e) {
+    console.log(e);
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the try-catch block is a good practice for error handling. However, it would be better to throw the error after logging it so that the calling function can handle it appropriately. Returning `undefined` when an error occurs might lead to unexpected behavior in the calling function if it's not prepared to handle this case.

```diff
  try {
    const compressedImage = await Image.compress(imagePath, {
      compressionMethod: 'auto',
      input: type,
      returnableOutputType: type
    });

    return compressedImage;
  } catch (e) {
    console.log(e);
+   throw e;
  }
```
Also, consider using `console.error` instead of `console.log` for logging errors as it provides stack trace which can be helpful for debugging.
```diff
- console.log(e);
+ console.error(e);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->